chain/store/store.go

```diff
@@ -98,6 +98,23 @@ func (cs *ChainStore) InitLedgerStoreWithGenesisBlock(genesisBlock *block.Block)
 			return 0, err
 		}
 
+		if config.Pruning {
+			refCountStartHeight, pruningStartHeight := cs.GetPruningStartHeight()
+			if refCountStartHeight == 0 && pruningStartHeight == 0 {
+				err := cs.SequentialPrune()
+				if err != nil {
+					return 0, err
+				}
+			} else if refCountStartHeight > 0 && pruningStartHeight > 0 {
+				err := cs.PruneStates()
+				if err != nil {
+					return 0, err
+				}
+			} else {
+				return 0, errors.New("get Start Height of pruning error")
```

Comment:
I think it's better to include `refCountStartHeight` and `pruningStartHeight` in error to help tracing problems

Reply:
i will add it.